README.md

```diff
@@ -291,43 +316,50 @@ Content-Type: application/pkcs7-mime
 
 The expiration headers in Package Headers section prescribe the 'useful lifetime' of the package, with UA optionally indicating the 'stale' state to the user and asking to upgrade, or automatically fetching a new one. While offline, the expiration may be ignored (not unlike Cache-Control: no-cache) but once user is online, the UA should verify both the certificate and if the package Content-Location contains an updated package (per Package Headers section) - and replace the package if necessary. In general, if the device is online and the package is expired, and the original location has updated package, the UA should obtain a new one (details TBD).
 
->Why is there a Content Index that specifies where each part is, and also MIME-like 'boundaries' that separate parts in the package?
+### Why is there a Content Index that specifies where each part is, and also MIME-like 'boundaries' that separate parts in the package?
 
 This is due to two main use cases of package loading:
 
 1. Loading the package from Web as part of the page or some other resource. In this case, the package is streamed from the server and **boundaries** allow to parse the package as it comes in and start using subresources as fast as possible. If the package has to be signed though, the package in its entirety has to be loaded first.
 2. Loading a (potentially large) package offline. In that case, it is important to provide a fast access to subresources as they are requested, w/o unpacking the package (it takes double the storage at least to unpack and significant time). Using direct byte-offset Content Index allows to directly access resources in a potentially large package.
 
->Is the certificate supplied by package a full chain to the known CA root?
+### Does the package supply a full chain of certificates to a known CA root?
 
-Not necessarily. To quote one of the Chrome security engineers, "because different devices have different sets of roots in their trust stores, it's not always the case that there is a single "correct" set of certificates to send that will work best for all clients. Instead, for compatibility, sites will sometimes send a set of certificates and rely on clients to dynamically fetch additional intermediates when needed". We suspect that to be an interesting issue since offline validation would need a full chain. Browsers (at least Chrome) are implementing automatic fetching of intermediate certificates but for this to work they have to be online. This will probably become a matter of best practices when creating the packaging format, possibly with validation by the packaging tool.
+Not necessarily. Different devices have different sets of roots in their trust
+stores, so there is not a single "correct" set of certificates to send that will
+work best for all clients. Instead, for compatibility, packages may need to
+include a set of certificates from which chains can be built to multiple roots,
```

Comment:
@sleevi is this true even if the publisher of the package obtains a separate TLS certificate for signing packages? Sort of similar to how code-signing certificates are separate form TLS ones.